desktop/src/org/limewire/util/FileUtils.java

```diff
@@ -224,70 +173,6 @@ public static boolean delete(File file, boolean moveToTrash) {
         }
     }
 
-    /**
-     * Moves the given file or directory to Trash.
-     *
-     * @param file The file or directory to move to Trash
-     * @return true on success
-     * @throws IOException if the canonical path cannot be resolved
-     *                     or if the move process is interrupted
-     */
-    private static boolean moveToTrashOSX(File file) {
-        try {
-            String[] command = moveToTrashCommand(file);
-            ProcessBuilder builder = new ProcessBuilder(command);
-            builder.redirectErrorStream();
-            Process process = builder.start();
-            consumeAllInput(process);
-            process.waitFor();
-        } catch (InterruptedException err) {
-            LOG.error("InterruptedException", err);
-        } catch (IOException err) {
-            LOG.error("IOException", err);
-        }
-        return !file.exists();
-    }
-
-    /**
-     * Consumes all input from a Process. See also
-     * ProcessBuilder.redirectErrorStream()
-     */
-    private static void consumeAllInput(Process p) throws IOException {
-        try (InputStream in = new BufferedInputStream(p.getInputStream())) {
-            byte[] buf = new byte[1024];
-            while (in.read(buf, 0, buf.length) >= 0) ;
-        }
-    }
-
-    /**
-     * Creates and returns the osascript command to move
-     * a file or directory to the Trash
-     *
-     * @param file The file or directory to move to Trash
-     * @return OSAScript command
-     * @throws IOException if the canonical path cannot be resolved
-     */
-    private static String[] moveToTrashCommand(File file) {
-        String path = null;
-        try {
-            path = file.getCanonicalPath();
-        } catch (IOException err) {
-            LOG.error("IOException", err);
-            path = file.getAbsolutePath();
-        }
-        String fileOrFolder = (file.isFile() ? "file" : "folder");
-        return new String[]{
-                "osascript",
-                "-e", "set unixPath to \"" + path + "\"",
-                "-e", "set hfsPath to POSIX file unixPath",
-                "-e", "tell application \"Finder\"",
-                "-e", "if " + fileOrFolder + " hfsPath exists then",
-                "-e", "move " + fileOrFolder + " hfsPath to trash",
-                "-e", "end if",
-                "-e", "end tell"
-        };
-    }
-
```

Comment:
Hopefully this works on macOS using just the Desktop.moveToTrash(), will test on my mac.